test/e2e/tests/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -205,7 +205,7 @@
   "SnapInterfaceController": "object",
   "SnapsRegistry": {
     "database": "object",
-    "lastUpdated": "object",
+    "lastUpdated": "number",
```

Comment:
OK I've had a look, this snapshot change seems valid.
Looking at the SnapRegistry this should be a number:
https://github.com/MetaMask/snaps/blob/main/packages/snaps-controllers/src/snaps/registry/json.ts#L86

I've had a look in the codebase, this field also is not used anywhere - so has low impact.

How this change occurred: I believe by installing the snap above also triggered an update to the `@metamask/snaps-controllers` package.

Reply:
This is actually unintended, and a symptom of a potential problem. This essentially means that during testing the Snaps registry was updated, but we only do that if users have Snaps installed to check if they need to be blocked. We don't need to do this for preinstalled Snaps. I will create a PR to get this fixed.

Reply:
https://github.com/MetaMask/metamask-extension/pull/23485

Reply:
OK nice find, I didn't know of this. Updated the branch and reran snapshots - all back to normal 👍🏾